stable/node-feature-discovery/templates/daemonset.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "node-feature-discovery.fullname" . }}
+  labels:
+{{ include "node-feature-discovery.labels" . | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+{{ include "node-feature-discovery.matchLabels" . | indent 6 }}
+  template:
+    metadata:
+      labels:
+{{ include "node-feature-discovery.labels" . | indent 8 }}
+    spec:
+      serviceAccountName: {{ template "node-feature-discovery.serviceAccountName" . }}
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      containers:
+        - env:
+          - name: NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          name: nfd-master
+          command:
+            - "nfd-master"
+          args:
+            - "--extra-label-ns=nvidia.com"
```

Comment:
Extra labels namespaces should be taken from values. Default can have the nvidia.com ns